oqsrtk/tests/soilprofile/test_proxies.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# =============================================================================
+#
+# Copyright (C) 2010-2017 GEM Foundation
+#
+# This file is part of the OpenQuake's Site Response Toolkit (OQ-SRTK)
+#
+# OQ-SRTK is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# OQ-SRTK is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# with this download. If not, see <http://www.gnu.org/licenses/>
+#
+# Author: Valerio Poggi
+#
+# =============================================================================
+
+import unittest
+import numpy as np
+
+from oqsrtk.soilprofile import proxies
+
```

Comment:
This is no good. Everything should be under the openquake umbrella. So something like

`from openquake.srtk.soilprofile import proxies`

Reply:
ok, now changed to openquake.srtk. It will be included with the new pull request